packages/cfd/src/features/Containers/trading-password-manager.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
```

Comment:
I suggest breaking this file into smaller components.

Reply:
Breaking of this file is not in the scope of this PR

Reply:
I think we should test for re-renders before we merge this, if you dont wanna break down to smaller components yet

Reply:
CU Card:
https://app.clickup.com/t/20696747/CFDS-1443

Reply:
@thisyahlen-deriv added the card for breaking down of component.
I can not do the breaking here as it is not the scope of PR.